evoxbench/test_suites/in1kmop.py

```diff
@@ -1,4 +1,4 @@
-from evoxbench.benchmarks import ResNet50DBenchmark, MobileNetV3Benchmark, TransformerBenchmark
```

Comment:
Do not remove the **TransformerBenchmark**.

Reply:
I see, thanks for correction!